### DISCUSSION TITLE
How to schedule immich-go

### DISCUSSION THREAD

**Discussion by archally298** [simulot/immich-go#536] [Q&A]
Hi all,

Does anyone have any advice on how to schedule immich-go?  When I run immich-go manually in the command line, it works fine, but when I try to run it in the User Scripts add-on in Unraid, I get the following error:

"panic: close of nil channel" 

Thanks!
-a

**Comment by redge76** [comment]
What command do you run in the user script ?

**Reply by archally298** [reply]
chmod +x file/path/to/immich-go

file/path/to/immich-go -server http://111.xxx.xx.xxx:xxxx -key <api key> upload /file/path/to/photos/

**Reply by redge76** [reply]
if it runs in the command line, you should have no need to add the chmod.
Does it run when you click "run script" ? Is this a "unraid" error or a immich-go error ?
This script  returns an "normal" error:
```
#!/bin/bash
/root/immich-go
```
I get:
```
Script location: /tmp/user.scripts/tmpScripts/immichgo/script
Note that closing this window will abort the execution of this script
immich-go 0.22.1, commit 35982819550c81ab615424cc10dd710d455e9952, built at 2024-10-15T19:49:03Z
. _ _ _ _ . _|_ _ _
|| | || | ||(_| | ─ (_|(_)
v 0.22.1 _)

ERROR | missing command upload|duplicate|stack|tool | time="2024-11-19T05:46.11 CET"
missing command upload|duplicate|stack|tool
```

**Reply by simulot** [reply]
add the option `-no-ui`

**Reply by simulot** [reply]
Add the rest of the command line in the bash script, add -no-ui  to avoid issues when running the command without console
